runtime-creativity/mypaint-brushes+1/autobuild/patches/0001-AOSCOS-bump-automake-to-1.18.patch

```diff
@@ -0,0 +1,29 @@
+From e801170402cc60dcbcb99bab7e14dc4aa873ad0a Mon Sep 17 00:00:00 2001
+From: ilikara <3435193369@qq.com>
+Date: Fri, 19 Sep 2025 20:53:38 +0800
+Subject: [PATCH] AOSCOS: bump automake to 1.18
+
+Signed-off-by: ilikara <3435193369@qq.com>
+---
+ autogen.sh | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/autogen.sh b/autogen.sh
+index ce41821..9102aba 100755
+--- a/autogen.sh
++++ b/autogen.sh
+@@ -7,9 +7,9 @@
+ # tools and you shouldn't use this script. Just call ./configure
+ # directly.
+ 
+-ACLOCAL=${ACLOCAL-aclocal-1.16}
++ACLOCAL=${ACLOCAL-aclocal-1.18}
```

Comment:
Might as well just remove the -1.18 suffix, so that we don't run into this issue again with 1.19.